CHANGELOG.md

```diff
@@ -3,7 +3,18 @@
 ## 0.7.0-beta.1
 
 * Use 1.7.0 of upstream SDK.
-* Use 1.7.0 of ASP.NET Core and HTTP instrumentations.
+* Use 1.7.0 of ASP.NET Core instrumentation.
+  * [Breaking change: option type renamed](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/1.6.0-rc.1).
```

Comment:
I'd remove this, as it only concerns user who are using this _in addition_ to our distro.